firestore/integration_test_internal/src/firestore_integration_test.cc

```diff
@@ -260,14 +261,10 @@ std::vector<std::string> FirestoreIntegrationTest::QuerySnapshotToIds(
   return result;
 }
 
-std::vector<MapFieldValue> FirestoreIntegrationTest::QuerySnapshotToValues(
-    const QuerySnapshot& snapshot) const {
-  std::vector<MapFieldValue> result;
-  for (const DocumentSnapshot& doc : snapshot.documents()) {
-    result.push_back(doc.GetData());
-  }
-  return result;
-}
+class AnotherTemporaryLintErrorClass {
+  // The below line should give a lint error.
+  AnotherTemporaryLintErrorClass(int t) {}
```

Comment:
⚠️ Lint warning: `Single-parameter constructors should be marked explicit.`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/firestore_integration_test.cc:266:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]"></hidden>